@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -190,18 +211,29 @@ export function checkReadyForPlacement(
  * @param textElement The text element of which the offset will computed. It must be a point
  * label with [[layoutStyle]] and [[bounds]] already computed.
  * @param offset The offset result.
+   @param scale The label scaling factor (1.0 original size).
+ * @param anchorPlacement The optional text placement parameter that overrides layout style
+ * settings stored in [[textElement.layoutStyle.horizontalAlignment]] and
+ * [[textElement.layoutStyle.verticalAlignment]]
  */
 function computePointTextOffset(
     textElement: TextElement,
     scale: number,
-    offset: THREE.Vector2 = new THREE.Vector2()
+    offset: THREE.Vector2 = new THREE.Vector2(),
+    anchorPlacement?: AnchorPlacement
```

Comment:
better remove this optional parameter if you're not using it.

Reply:
New version removes this function entirely so marking as solved.